developer/developer-guide.md

```diff
@@ -1,3 +1,872 @@
 \include{metadata.md}
```

Comment:
There is something weird going on with the generated website: every link goes to a generated directory page, instead of the index.html page itself.
However, this could also be due to the fact that I'm reading the website from my local disk, and not via a webserver.

Reply:
I believe that's default behaviour for local files.

Reply:
Yes probably. You can ignore this remark.